frontends/common/options.cpp

```diff
@@ -145,8 +180,8 @@ FILE* CompilerOptions::preprocess() {
 #else
         std::string cmd("cpp");
 #endif
-        cmd += cstring(" -undef -nostdinc -I") +
-                p4includePath + " " + preprocessor_options + " " + file;
+        cmd += cstring(" -undef -nostdinc -I") + (isv1() ? p4_14includePath : p4includePath)
```

Comment:
Shouldn't the p4includePath be appended at the end?
This way we cannot override it with -I.